odm2api/ODM2/services/readService.py

```diff
@@ -554,11 +554,12 @@ def getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=N
         q = self._session.query(Affiliations)
 
         if ids: q = q.filter(Affiliations.AffiliationID.in_(ids))
-        if orgcode: q = q.join(Organizations).filter(Organizations.OrganizationCode.ilike(orgcode))
-        if personfirst or personlast:
-            q = q.join(People)
-            if personfirst: q = q.filter(People.PersonFirstName.ilike(personfirst))
-            if personlast: q = q.filter(People.PersonLastName.ilike(personlast))
+        if orgcode: q = q.join(Affiliations.OrganizationObj).filter(
```

Comment:
@emiliom Is this what you're looking for? I still have to do the join... Can't find anything that simply filters straight up from relationship.

Reply:
I got the sample from https://github.com/ODM2/ODM2PythonAPI/blob/master/odm2api/ODM2/services/readService.py#L421